generators/client/templates/angular/src/main/webapp/app/layouts/main/_main.component.ts

```diff
@@ -50,6 +50,15 @@ export class <%=jhiPrefixCapitalized%>MainComponent implements OnInit {
                     params = destinationEvent.params;
                     destinationData = destinationEvent.data;
                     destinationName = destinationEvent.url[0].path;
+                } else {
+                    destinationEvent = event.state.root.children[0];
```

Comment:
In fact I don't rly know why we need this, isn't the guard sufficient ? Like here https://angular.io/docs/ts/latest/guide/router.html ("Guard the admin feature")